src/main/java/uk/gov/hmcts/reform/translate/model/DictionaryRequest.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.translate.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+
+import java.util.Map;
+
+@Getter
+public class DictionaryRequest {
```

Comment:
I think this class is basically doing the same as `Dictionary`? Could we perhaps just re-use `Dictionary` now it is generic enough?

Reply:
yes :) The idea was moving it to dictionary . si